### ISSUE TITLE
Browsing changelog by project is only possible one release at a time

### ISSUE THREAD

**Issue by LittleEndu** [ppy/osu-web#4985]
When you first open [changelog](https://osu.ppy.sh/home/changelog) you see many many releases and when you scroll all the way down you get a ``Show more`` button. But when you change to only view stable/lazer/web changelogs, you can only see one release and have to press the previous button to see the previous release which loads a new page (there's no AJAX).

**Comment by peppy** [comment]
So you'd rather see a feed per build?

**Comment by LittleEndu** [comment]
I mean if you right now go to changelog and click web it takes you to https://osu.ppy.sh/home/changelog/web/2019.919.0 and that is the only release on that page. 

There is no way to see all the web releases and only web releases without also seeing releases from lazer, stable, and CE by just staying at the general changelog page.

**Comment by Pennek** [comment]
have to chime in here and say that this would be preferable for osu!wiki work as well. auto-updating the feed to show more entries if the bottom is reached would also be a huge quality of life update. browsing the changelog for historical uses is painful currently

**Comment by Joehuu** [comment]
Can be done with https://osu.ppy.sh/home/changelog?stream=web, but that's not exposed in the UI.

**Comment by MilkyIQ** [comment]
I realize this is quite an old issue but I'd like to chime in and say that I think a search function (just for dates not for text) or some sort of index would be much more helpful than an auto-update even if its a major PITA. The changelog would be an invaluable resource for historical record-keeping, but it's practically impossible to use for older dates unless you want to go through every single date of a month in the URL

Any progress on this would be massively appreciated

**Comment by peppy** [comment]
Wonder how reusable the news page's history UI is?

![20210312 152728 (Safari app)](https://user-images.githubusercontent.com/191335/110901303-7c85eb00-8347-11eb-9c66-e505848ed78d.png)

Could work quite well here too.

**Comment by LittleEndu** [comment]
Yeah, but I originally was looking for a specific change in one of the releases not for a specific release. Having an option to see all releases in a time period would be nice but just seeing the release number on the sidebar doesn't help.

**Comment by peppy** [comment]
That sounds like a super edge case. How about you just send me an email asking rather than requiring a full search system to be implemented just for a simlpe request?

**Comment by LittleEndu** [comment]
No, what I wanted can already be done with what Joehuu said (it just doesn't have UI)

What MilkyIQ chimed in with is a separate issue entirely imo
